src/system/catalog-sync.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Catalog Sync
+---
+
+Magento exports catalog data into tables using indexers. This process is automatically [triggered by several events]({% link system/index-management-events.md %}), such as changes to a product's price or inventory levels. Magento then syncs that catalog data to a central storage hub for use by various SaaS services, such as [Product Recommendations]({% link marketing/product-recommendations.md %}). The **Catalog Sync** dashboard allows you to peek into the Magento to SaaS syncing process and view the current status.
```

Comment:
```suggestion
Magento compiles catalog data into tables using indexers. This process is automatically [triggered by several events]({% link system/index-management-events.md %}), such as changes to a product's price or inventory levels. Magento then syncs that catalog data to an endpoint where it can then be used by various SaaS services, such as [Product Recommendations]({% link marketing/product-recommendations.md %}). The **Catalog Sync** dashboard allows you to observe and manage the Magento to SaaS syncing process.
```